CHANGES.rst

```diff
@@ -13,6 +13,23 @@ Changelog
 
 .. towncrier release notes start
 
+4.6.1 (2022-10-14)
+Features
+--------
+
+- Add limited v1 support for legacy roles specific for use in galaxy.ansible.com
```

Comment:
These changelog entries should have gone into 4.6.0. I'm not sure how they ended up here, but can you move them under 4.6.0?

We don't have any changelog entries for 4.6.1. Maybe just add a manual note saying we're upgrading ansible-lint